CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.4.1
+- Add `willShowViewControllerSignal` to `UINavigationController` that reflects UINavigationControllerDelegate's `navigationController(_:willShow:animated:)` delegate method.
```

Comment:
Would be great if you bumped the project and pod file version as well.

Reply:
@mansbernhardt Sorted!